include/boot.h

```diff
@@ -43,149 +43,149 @@ extern const char _start[];
 extern volatile u32 lz_stack_canary;
 
 typedef struct __packed sl_header {
-	u16 lz_entry_point;
-	u16 bootloader_data_offset;
-	u16 lz_info_offset;
+    u16 lz_entry_point;
+    u16 bootloader_data_offset;
+    u16 lz_info_offset;
 } sl_header_t;
 extern sl_header_t sl_header;
 
 typedef struct __packed lz_info {
-	u8  uuid[16]; /* 78 f1 26 8e 04 92 11 e9  83 2a c8 5b 76 c4 cc 02 */
-	u32 version;
-	u16 msb_key_algo;
-	u8  msb_key_hash[64]; /* Support up to SHA512 */
+    u8  uuid[16]; /* 78 f1 26 8e 04 92 11 e9  83 2a c8 5b 76 c4 cc 02 */
+    u32 version;
+    u16 msb_key_algo;
+    u8  msb_key_hash[64]; /* Support up to SHA512 */
 } lz_info_t;
 extern lz_info_t lz_info;
 
 /* The same as TPML_DIGEST_VALUES but little endian, as event log expects it */
 typedef struct __packed ev_log_hash {
-	u32 count;
-	u16 sha1_id;
-	u8 sha1_hash[20];
-	u16 sha256_id;
-	u8 sha256_hash[32];
+    u32 count;
+    u16 sha1_id;
+    u8 sha1_hash[20];
+    u16 sha256_id;
+    u8 sha256_hash[32];
 } ev_log_hash_t;
 
 /* Fences */
-#define mb()		asm volatile("mfence" : : : "memory")
-#define rmb()		asm volatile("lfence" : : : "memory")
-#define wmb()		asm volatile("sfence" : : : "memory")
-#define barrier()	asm volatile("" : : : "memory")
+#define mb()        asm volatile("mfence" : : : "memory")
+#define rmb()       asm volatile("lfence" : : : "memory")
+#define wmb()       asm volatile("sfence" : : : "memory")
+#define barrier()   asm volatile("" : : : "memory")
 
-#define smp_rmb()	barrier()
-#define smp_wmb()	barrier()
-#define smp_mb()	mb()
+#define smp_rmb()   barrier()
+#define smp_wmb()   barrier()
+#define smp_mb()    mb()
```

Comment:
Depending how often smp_mb() gets used, we might consider switching to the relaxed form which a locked operation against the stack.  It's lower overhead than `mfence`, but a longer instruction so is a minor space tradeoff.

Reply:
For this PR I think such a change is out of scope. A subsequent PR can be posted where this and the ROUNDUP macro could be addressed, along with some other optimizations.